### DISCUSSION TITLE
503 Service Unavailable on forwarded ports

### DISCUSSION THREAD

**Discussion by ImAnAutie** [community/community#4830] [Codespaces] [answered]
Hi there,
I'm accessing my codespaces via the browser (a mix of chrome 91.0.4472.164 on Linux and the chromium based Edge on windows) and I'm now consistently getting 503 when visiting the forwarded ports regardless of private/public.
It also seems to be ignoring the public/private setting as it's redirecting through the usual private auth flow even when it's showing in the port forwarding tab as public.

Here's a few request ID's from one of my codespaces

Request ID: 3c225dce5d137304b60a12041616b07e
Request ID: d2a47477fac1938c2846919aa82ec1b5

Any ideas? they are all express js apps behind the scenes and curl to them from the codespaces shell appears to work fine.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @ImAnAutie thanks for reaching out. 

Can you please create a support ticket for this by going here: https://support.github.com/contact

That would help our engineering team get more information if needed and investigate further :)

**Reply by ImAnAutie** [reply]
Wilco :)

**Comment by treeder** [comment]
I'm having a similar issue, when I set a port to public, it will sometimes flip back to private even though it stills says public in the ports tab.

**Reply by ImAnAutie** [reply]
I've noticed that happen's sometimes when the service running on the port hangs up is is restarted.
